src/moin/converters/html_out.py

```diff
@@ -11,14 +11,16 @@
 
 import re
 
-from flask import request
 from flask import current_app as app
 from emeraldtree import ElementTree as ET
+from werkzeug.urls import url_encode
 
 from moin import wikiutil
 from moin.i18n import _
+from moin.items import Item
```

Comment:
this is causing a circular import, can anyone assist in fixing:

>>> from moin.items import Item
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bylsmad/PycharmProjects/moin/src/moin/items/__init__.py", line 42, in <module>
    from moin.storage.middleware.protecting import AccessDenied
  File "/home/bylsmad/PycharmProjects/moin/src/moin/storage/middleware/protecting.py", line 63, in <module>
    class ProtectingMiddleware:
  File "/home/bylsmad/PycharmProjects/moin/src/moin/storage/middleware/protecting.py", line 64, in ProtectingMiddleware
    from .indexing import parent_names
  File "/home/bylsmad/PycharmProjects/moin/src/moin/storage/middleware/indexing.py", line 82, in <module>
    from moin.converters import default_registry
  File "/home/bylsmad/PycharmProjects/moin/src/moin/converters/__init__.py", line 60, in <module>
    load_package_modules(__name__, __path__)
  File "/home/bylsmad/PycharmProjects/moin/src/moin/utils/pysupport.py", line 36, in load_package_modules
    importlib.import_module(module)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/bylsmad/PycharmProjects/moin/src/moin/converters/html_out.py", line 20, in <module>
    from moin.items import Item
ImportError: cannot import name 'Item' from partially initialized module 'moin.items' (most likely due to a circular import) (/home/bylsmad/PycharmProjects/moin/src/moin/items/__init__.py)